src/js/tech/html5.js

```diff
@@ -549,9 +549,19 @@ class Html5 extends Tech {
    * @method setSrc
    */
   setSrc(src) {
+    let loadstartlistener = Html5.prototype.loadStartListener_;
+
+    this.off(this.el_, 'loadstart', loadstartlistener);
+    this.one(this.el_, 'loadstart', () => this.one(this.el_, 'loadstart', loadstartlistener));
```

Comment:
Here's the scenario:

1. Call setSrc()
1. Video element fires `loadstart`
1. Call setSrc()

loadStartListener_() would be removed on the second call to setSrc() but the inline listener that triggers the loadstartlistener on the next `loadstart` *would not*. So, the very next `loadstart` would run loadstartlistener.